pirant/models.py

```diff
@@ -0,0 +1,35 @@
+import colander
```

Comment:
Isn't it better to keep codes as modular as possible? 
Few advantages I see keeping codes modular:
1. keeping different models in separate files reduces chances of merge conflict. Otherwise, for new APIs, everyone will be working on same file, increasing chances of conflicts. 
2. Increase readability and understandability if someone new jumps into contribution.
As you mentioned that more pythonic is to have all in one file. Does having separate files causes latency or some sort of issues?

Reply:
I understand your point of view. Actually, a couple of years ago I thought the same thing and wrote one module per class. This way is not better nor worse, it's just different. The idea is to have all the classes that are related to each other (and are dependent on each other) into the same module. One advantage is that the project "looks" cleaner as there aren't so many files. As for the conflicts, as long as two people don't work on the same class, there won't be merge conflicts (if they touch the same class, they will have merge conflicts either way).

Having separate files does not have any latency problem, but it complicates importing modules, while if you have all the classes in one module, you don't even need to import them.

Reply:
Another big plus point on putting related classes in the same module is that pirant is a python library, and python programmers expect things to be organized like this because it's the "pythonian" way. It's the same reason why I would NEVER do this in Java or C++ (if it was even possible). Java should look like Java, and Python should look like Python ;)